### PR TITLE
Improve mutation generated code

### LINKED ISSUE #40: Improve Mutation templates to have a pre-ready Mongoose save/update
The generated templates should have a code to save or edit the record in `MutationAdd` and `MutationEdit`.

### PR DESCRIPTION
**Pull request by sibelius**
Fixes #40.

### CODE REVIEW

File: packages/generator/src/mutation/templates/MutationAddWithSchema.js.template [hunk 1]
```diff
@@ -37,10 +37,17 @@ export default mutationWithClientMutationId({
       <%_ } -%>
     } = args;
 
+    // Create new model
+    const <%= camelCaseName %> = await new <%= name %>({
+        <%_ for (field of schema.fields) { -%>
```
Comment (lucasbento) [inline packages/generator/src/mutation/templates/MutationAddWithSchema.js.template]: The indentation is with tabs instead of 2 spaces.

Comment (lucasbento) [inline packages/generator/src/mutation/templates/MutationAddWithSchema.js.template]: These fields should also have a value, can be `'Example value'` for now and the user fulfils them later.

File: packages/generator/src/mutation/templates/MutationEditWithSchema.js.template [hunk 2]
```diff
@@ -53,6 +53,13 @@ export default mutationWithClientMutationId({
       throw new Error('Invalid <%= camelCaseName %>Id');
     }
 
+    // Edit model
+    await <%= camelCaseName %>.update({
+        <%_ for (field of schema.fields) { -%>
```
Comment (lucasbento) [inline packages/generator/src/mutation/templates/MutationEditWithSchema.js.template]: Same thing here, missing values and change indentation to 2 spaces.

File: packages/generator/src/mutation/templates/MutationAddWithSchema.js.template (hunk 1 above)
Comment (sibelius) [inline packages/generator/src/mutation/templates/MutationAddWithSchema.js.template]: the values came from graphql mutation input

**Pull request merged**